core/src/main/java/org/lflang/generator/c/CCmakeGenerator.java

```diff
@@ -504,9 +505,9 @@ private static String setUpMainTargetRp2040(
     code.pr("pico_sdk_init()");
     code.newLine();
     code.pr("add_subdirectory(core)");
-    code.pr("target_link_libraries(core PUBLIC pico_stdlib)");
-    code.pr("target_link_libraries(core PUBLIC pico_multicore)");
-    code.pr("target_link_libraries(core PUBLIC pico_sync)");
+    code.pr("target_link_libraries(reactor-c PUBLIC pico_stdlib)");
+    code.pr("target_link_libraries(reactor-c PUBLIC pico_multicore)");
+    code.pr("target_link_libraries(reactor-c PUBLIC pico_sync)");
```

Comment:
The library in `core` was renamed from `core` to `reactor-c`, breaking single-threaded rp2040 support. This is bugfix.

There might be a cleaner way to link the pico libraries to the platform files (perhaps linking from within core/CMakeLists.txt), but that would probably be better to handle in a separate PR.